hera_cal/io.py

```diff
@@ -1447,8 +1464,8 @@ def __init__(self, input_data, read_metadata=True, check=False, skip_lsts=False)
     def _adapt_metadata(self, info_dict, skip_lsts=False):
         '''Updates metadata from read_hera_hdf5 to better match HERAData. Updates info_dict in place.'''
         info_dict['data_ants'] = sorted(info_dict['data_ants'])
-        info_dict['antpairs'] = sorted(info_dict['bls'])
-        info_dict['bls'] = sorted(set([ap + (pol, ) for ap in info_dict['antpairs'] for pol in info_dict['pols']]))
+        info_dict['antpairs'] = info_dict['bls']
+        info_dict['bls'] = list(set([ap + (pol, ) for ap in info_dict['antpairs'] for pol in info_dict['pols']]))
```

Comment:
I think the sorted() is an unnecessary time-waste. But I've added it back in, in case it affects something I'm not aware of. I can't remember the exact reason I removed it originally.